market_maker_keeper/erisx_market_maker_keeper.py

```diff
@@ -158,6 +159,12 @@ def _thread_refresh_order_book(self):
                 with self._lock:
                     orders = self.get_orders_function()
 
+                # remove canceled orders from local orders instance
+                with self._lock:
+                    for order in orders:
+                        if order.order_id in orders_already_cancelled_before:
+                            orders.remove(order)
```

Comment:
Since we're removing cancelled orders here in the order book manager, can we add placed orders here too?

Reply:
We could, but it would then be higher up in the call stack, I was removing the cancelled orders here so as to avoid having to subclass any additional methods. That being said, I can work on moving it into here to keep things consistent.